x-pack/plugins/cloud_security_posture/common/schemas/csp_finding.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// TODO: this needs to be defined in a versioned schema
+import type { EcsEvent } from '@kbn/logging';
+import type { CspRuleMetadata } from './csp_rule_metadata';
+
+export interface CspFinding {
```

Comment:
This file needed to be shared in the first place, moved into the common folder so the server will have access to it also.
 
for now i have not deleted the original type which is under `public/pages/findings/types`. due to it being very commonly used thought the system and removing it should probably be a separate pr.

will open a task and update this comment after

Reply:
I strongly suggest doing this as part of this task. Having two sources of truth for the data leads to confusion and potentially changing one without the other, even if this is short-lived. This should be an easy fix since this is just a duplication of the interface, just change the imports.